config/routes.rb

```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
   resources :posts
+  resources :advertisements
```

Comment:
cruft? also, it opens up our app to routes that aren't handled... the system will try to route traffic to routes that don't have actions... so you'll 500 instead of 404 (404 is more graceful)